src/client/datascience/notebookStorage/nativeEditorStorage.ts

```diff
@@ -220,11 +216,6 @@ export class NativeEditorStorage implements INotebookStorage {
             // eslint-disable-next-line @typescript-eslint/no-explicit-any
             return (notebookData.metadata.language_info.codemirror_mode as any).version;
         }
-        // Use the active interpreter if allowed
-        if (this.extensionChecker.isPythonExtensionInstalled) {
```

Comment:
This is totally unnecessary, we're just storing this in metadata for language, today we generate all of this information from the kernel, hence we have more accurate information than this.

Reply:
Also this was more stuff that slowed down notebooks, unnecessary when creating blank notebooks.